docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -7,74 +7,221 @@
-in the generated date field of the destination index. The reason is that the `yyyy`
-value can be identified incorrectly as `epoch_millis`. As a workaround, using the 
-API, you may define a custom destination index data format mapping prior to starting 
-the {dataframe-transform}. For example:
-
-[source, json]
-------------------------------------------------------------
-"mappings" : {
-        "properties" : {
-            "custom_date" : { 
-            "type" : "date", 
-            "format": "yyyy"
-            }
-          }
-        }
-------------------------------------------------------------
+[[df-consistency-limitations]]
+=== {cdataframe-cap} consistency does not account for deleted or updated documents
+
+While the process for {cdataframe-transforms} allows the continual recalculation 
+of the {dataframe-transform} as new data is being ingested, it does also have 
+some limitations.
+
+Changed entities will only be identified if their time field 
+has also been updated and falls within the range of the action to check for 
+changes. This has been designed in principal for, and is suited to the use case 
+where new data is given a timestamp for the time of ingest. 
+
+If the indices that fall within the scope of the source index pattern are 
+removed, for example when deleting historical time-based indices, then the 
+composite aggregation performed in consecutive checkpoint processing will search 
+over different source data, and entities that only existed in the deleted index 
+will not be removed from the {dataframe} destination index.
+
+Depending on your use case, you may wish to recreate the {dataframe-transform} 
+entirely after deletions. Alternatively, if your use case is tolerant to 
+historical archiving, you may wish to include a max ingest timestamp in your 
+aggregation. This will allow you to exclude results that have not been recently 
+updated when viewing the {dataframe} destination index.
+
 
 [float]
-[[df-aggresponse-limitations]]
-=== Aggregation responses may be incompatible with destination index mappings
+[[df-deletion-limitations]]
+=== Deleting a {dataframe-transform} does not delete the {dataframe} destination 
+index or {kib} index pattern
+
+When deleting a {dataframe-transform} using `DELETE _data_frame/transforms/index` 
+neither the {dataframe} destination index nor the {kib} index pattern, should 
+one have been created, are deleted. These objects must be deleted separately.
+
+[float]
+[[df-aggregation-page-limitations]]
+=== Handling dynamic adjustment of aggregation page size
+
+During the development of {dataframe-transforms}, control was favoured over 
+performance. In the design considerations, it is preferred for the 
+{dataframe} to take longer to complete quietly in the background rather than to 
+finish quickly and take precedence in resource consumption.
+
+Composite aggregations are well suited for high cardinality data enabling 
+pagination through results. If a parent 
```

Comment:
just "circuit breaker", without the "parent" (the concept of parent circuit breaker exceptions doesn't fit in this context)